### ISSUE TITLE
Incorporate schema.org types: EOProgram, Course, CourseInstance

### ISSUE THREAD

**Issue by kimdhamilton** [w3c-ccg/vc-ed-models#22]
In addition to schema.org EOCred, which we've discussed, the following schema.org types look very useful:
- EOProgram
- Course
- CourseInstance

## Example Scenario

- VC using schema.org
- Representing that a `Person` has earned an `EOCred`
- Additionally containing details about the program (`EOProgram`?) and/or `Course`/`CourseInstance`s associated with the program.

Note: Navigationally, I can get from `EOProgram` to `EOCred` via `educationalCredentialAwarded`, but there's no pre-defined backwards link.

## Options to express this

1. `Person` `hasCredential` `EOCred` as a result of `EOProgram`

<img width="1166" alt="Screen Shot 2020-11-14 at 11 22 49 AM" src="https://user-images.githubusercontent.com/3945683/99207878-fdefb700-2773-11eb-8f16-b1a5ce98b06f.png">

> schema.org impact: needs reverse nav from EOCred to EOProgram

2. array of two claims about `Person`:
    - `Person` `hasCredential` `EOCred`, and
    - `Person` `hasCompleted`/`completedProgram` (name TBD)  `EOProgram` 

<img width="1128" alt="Screen Shot 2020-11-14 at 12 34 38 PM" src="https://user-images.githubusercontent.com/3945683/99207886-021bd480-2774-11eb-915e-1606ffc069b1.png">

> schema.org impact: needs relationship from Person to EOProgram

### Discussion

#2 would result in a flatter structure, and the more complex relationships could still be determined (dotted line) out of the context of a single VC, for example, by separate knowledge of the entities and relationships.

Would either be a candidate for a new addition to schema.org, given that this is pretty common in VC EDU scenarios.

**Comment by anthonycamilleri** [comment]
I would say that this isn't a new instance - it describes the 'concept' and 'instance' that aligns with what we're calling Concept 3.

**Comment by kimdhamilton** [comment]
Right, this isn't a new use case; I just want to work through how it would work with schema.org types. I think EOCred, EOProgram, and Course will be especially interesting types to factor in. 

I think this is 1 combined with 3: the learner completed a program (1) and, as a result, attained a credential (3)

**Comment by kimdhamilton** [comment]
I updated the title to make that clearer
